extensions/sql-database-projects/src/dialogs/deployDatabaseQuickpick.ts

```diff
@@ -70,6 +70,50 @@ export async function launchDeployAppIntegrationQuickpick(project: Project): Pro
 	};
 }
 
+async function launchEulaQuickPick(baseImage: string): Promise<boolean> {
+	let eulaAccepted: boolean = false;
+	const baseImages = uiUtils.getDockerBaseImages();
+	const imageInfo = baseImages.find(x => x.name === baseImage);
+	const agreementInfo = imageInfo?.agreementInfo;
+	if (agreementInfo) {
+		const openEulaButton: vscode.QuickInputButton = {
+			iconPath: new vscode.ThemeIcon('link-external'),
+			tooltip: constants.openEulaString
+		};
+		const quickPick = vscode.window.createQuickPick();
+		quickPick.items = [{ label: constants.yesString },
+		{ label: constants.noString }];
+		quickPick.placeholder = uiUtils.getAgreementDisplayText(agreementInfo);
```

Comment:
I probably wouldn't use placeholder for this (since if they start typing in it then it'll disappear). Title seems more appropriate